spec/controllers/api/jobs_controller_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+RSpec.describe Api::JobsController, type: :controller do
+  describe "POST#create" do
+    it "can create a new job with no company or date" do
+      job = {
```

Comment:
didnt use factory girl for this test since its not being added to the DB directly, just manually creating through the post route since it needs to be in this format.